src/app/components/forms/game/score-table/score-table.vue

```diff
@@ -22,13 +23,15 @@ export default class FormGameScoreTable extends mixins(Wrapper) {
 	modelSaveHandler = $saveGameScoreTable;
 
 	readonly GameScoreTable = GameScoreTableModel;
+	readonly DirectionAscend = GameScoreTableSorting.DirectionAsc;
+	readonly DirectionDescend = GameScoreTableSorting.DirectionDesc;
 
 	onInit() {
 		this.setField('game_id', this.game.id);
 
 		if (this.method === 'add') {
 			this.setField('unique_scores', true);
-			this.setField('scores_sorting_direction', GameScoreTableModel.SORTING_DIRECTION_DESC);
+			this.setField('scores_sorting_direction', this.DirectionDescend);
```

Comment:
outside of templates, use the enum directly instead of the one on the class instance. it'll be easier to convert these components in the future then